proto/buf/alpha/registry/v1alpha1/recommendation.proto

```diff
@@ -40,6 +70,20 @@ service RecommendationService {
   rpc RecommendedRepositories(RecommendedRepositoriesRequest) returns (RecommendedRepositoriesResponse);
   // RecommendedTemplates returns a list of recommended templates.
   rpc RecommendedTemplates(RecommendedTemplatesRequest) returns (RecommendedTemplatesResponse);
+  // ListRecommendedRepositories returns a list of recommended repositories that user have access to.
+  rpc ListRecommendedRepositories(ListRecommendedRepositoriesRequest) returns (ListRecommendedRepositoriesResponse);
+  // ListRecommendedTemplates returns a list of recommended templates that user have access to.
+  rpc ListRecommendedTemplates(ListRecommendedTemplatesRequest) returns (ListRecommendedTemplatesResponse);
+  // ListRecommendations returns a list of recommendations of a resource type.
+  rpc ListRecommendations(ListRecommendationsRequest) returns (ListRecommendationsResponse);
```

Comment:
Do we need this if we have the other two?

Reply:
the other two are for normal user to get the recommendation list, this `ListRecommendations` is for server admin to list and manage the recommendations, so it might be better to have both of them, or we will need to expose more information to normal user so that server admin can use those information to manage the recommendations.

Reply:
What information do we need to expose to the server_admin that we don't want to expose to the user? I'm confused, can we not use both the List methods for the server admin too?

Reply:
the `recommendation_id`, I don't think we don't want to expose this to the user, but this is not needed. But I guess it's fine to just include this in the other two endpoints